pytest.ini

```diff
@@ -2,3 +2,4 @@
 markers =
     online: marks tests that use online resources (deselect with '-m "not online"')
     slow: marks tests that are slow (deselect with '-m "not slow"')
+    vpn: marks tests that use the UVic VPN (deselect with '-m "not vpn"')
```

Comment:
Is there any circumstance that a test will be marked `online` but not `vpn`?

Reply:
I don't believe so. I just put that there as a reminder to myself since I hope to get rid of that.